cmd/node/config/enableEpochs.toml

```diff
@@ -239,6 +239,9 @@
     # AlwaysSaveTokenMetaDataEnableEpoch represents the epoch when the token metadata is always saved
     AlwaysSaveTokenMetaDataEnableEpoch = 2
 
+		# RuntimeCodeSizeFixEnableEpoch represents the epoch when the code size fix in the VM is enabled
```

Comment:
indentation ?

Reply:
yup, please use space instead of tabs

Reply:
Replaced with spaces inside test strings.